setup.py

```diff
@@ -1,20 +1,14 @@
-import os
-import sys
-import platform
-import subprocess
 from setuptools import find_packages
 from skbuild import setup
 
 from version import version
 
-long_description = open('README.rst').read()
-
 setup(
     name='h3',
     version=version,
     description=
     'Python bindings for H3, a hierarchical hexagonal geospatial indexing system developed by Uber Technologies',
-    long_description=long_description,
+    long_description='long_description',
```

Comment:
Probably should include the contents of README.md as the long description, as before